crates/starknet-devnet/benches/mint_bench.rs

```diff
@@ -0,0 +1,34 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use tokio::runtime::Runtime;
+
+use crate::common::background_devnet::BackgroundDevnet;
+
+#[path = "../tests/common/mod.rs"]
+pub mod common;
+
+static DUMMY_ADDRESS: u128 = 1;
+static DUMMY_AMOUNT: u128 = 1;
+
+async fn mint_iter(capacity: &str) {
+    let devnet = BackgroundDevnet::spawn_with_additional_args(&["--state-archive-capacity", f])
+        .await
+        .expect("Could not start Devnet");
+
+    for _n in 1..=100 {
```

Comment:
I would change 100 to 5000 and keep this bench test but I would remove [crates/starknet-devnet/tests/test_memory.rs](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/439/files#diff-261995e29036d91efa5b511ece3ab6e46f8ead5b2573ac46f7da7b9a2edf06cc) since it will not work as expected.

Also, I would mention in the documentation that the developer should use an external tool to monitor memory.

Reply:
ok I have done that as requested.